contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Strategizable } from "../governance/Strategizable.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract OETHHarvesterSimple is Strategizable {
+    using SafeERC20 for IERC20;
+
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    mapping(address => bool) public supportedStrategies;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event Harvested(address token, uint256 amount);
+    event SupportedStrategyUpdated(address strategy, bool status);
+
+    ////////////////////////////////////////////////////
+    /// --- CONSTRUCTOR
+    ////////////////////////////////////////////////////
+    constructor(address _governor, address _strategist) {
+        _setStrategistAddr(_strategist);
+        _changeGovernor(_governor);
+    }
+
+    ////////////////////////////////////////////////////
+    /// --- MUTATIVE FUNCTIONS
+    ////////////////////////////////////////////////////
+    function harvestAndTransfer(address _strategy) external {
+        _harvestAndTransfer(_strategy);
+    }
+
+    function harvestAndTransfer(address[] calldata _strategies) external {
+        for (uint256 i = 0; i < _strategies.length; i++) {
+            _harvestAndTransfer(_strategies[i]);
+        }
+    }
+
+    function _harvestAndTransfer(address _strategy) internal {
+        // Ensure strategy is supported
+        require(supportedStrategies[_strategy], "Strategy not supported");
+
+        // Harvest rewards
+        IStrategy(_strategy).collectRewardTokens();
+
+        // Cache reward tokens
+        address[] memory rewardTokens = IStrategy(_strategy)
+            .getRewardTokenAddresses();
+        for (uint256 i = 0; i < rewardTokens.length; i++) {
+            // Cache balance
+            uint256 balance = IERC20(rewardTokens[i]).balanceOf(address(this));
+            if (balance > 0) {
+                // Transfer to strategist
+                IERC20(rewardTokens[i]).safeTransfer(strategistAddr, balance);
+                emit Harvested(rewardTokens[i], balance);
```

Comment:
nit: Let's emit the strategy address here as well?

Reply:
Yes good idea, done in this commit: [0e8ab277](https://github.com/OriginProtocol/origin-dollar/pull/2333/commits/0e8ab277eb269cbbe30175dc2cdb0084b1d5dc83)